.github/workflows/retro.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8"]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
@PProfizi could you please either make a doc or setup a meeting (maybe with a slide on which pipelines are available) to decide when each pipeline is ran?